tests/unit/test_stages/test_base_stage.py

```diff
@@ -151,6 +147,23 @@ def test_dashboard_view(self, workgroups, target_students, state, stats):
         self.assertDictionaryEqual(args[1], expected_context)
 
     @ddt.data(
+        (make_stats(0.4, 0.3, 0.4), (40, 30, 40)),
+        (make_stats(0.1, 0.1, 0.8), (80, 10, 10)),
+        (make_stats(None, None, None), (None, None, None))
+    )
+    @ddt.unpack
+    def test_make_human_stats(self, stats, human_stats_data):
+        stats_order = (StageState.NOT_STARTED, StageState.INCOMPLETE, StageState.COMPLETED)
+        expected_human_stats = BaseGroupActivityStage.make_human_stats(stats)
```

Comment:
@jbzdak I used `expected_*` to denote expected values, and `actual_*` for actual results of function call/request/whatever else is tested. So it's quite the opposite, this var should be `actual_human_stats` and `human_stats_data` should be `expected_human_stats`